### ISSUE TITLE
Emeritus status and recognition

### ISSUE THREAD

**Issue by markwhiting** [crowdresearch/collective#5]
Contributors to our community have not been publicly recognized in a way that outlives their significant participation. This is problematic because one of the key benefits of being part of this project is its potential to help people move forward in their careers, and that often requires public documentation.

### My proposal
This proposal has two elements:
1. **create an emeritus status** formalized as participants who have made substantial contributions in a particular area in the past but have stepped away from that line of work. Specifically, people who have been co-authors but have not been part of more recent research efforts since their co-authorship would be given emeritus author status. Alternatively people who have committed to engineering efforts and have not contributed in development since the last publication or milestone would be given emeritus engineering status. Other types of emeritus status would be developed for other types of contribution (e.g., design, governance). Participants with emeritus status would: be listed on our public web presence, be relieved of ongoing responsibilities, and remain active in slack for as long as they prefer. Emeritus status would be automatically issued on a per category basis. The public listing of the status could be optionally withdrawn if the participant so chose. Participants who are emeritus would be able to reengage the community as any other participant if they chose to continue their work with us.
2. **create a page to document community members emeritus status** and make it available to the public. In the short term, this could be a list of people put on a static page as part of our web presence. Individuals could ask to be removed if they were uncomfortable with being listed publicly.

---
This imports https://github.com/crowdresearch/daemo/issues/1030 and updates it to the newer proposal template.

---
**Use comments to share your response** or use emoji 👍 to show your support. **To officially join in**, comment `I will help.` **To break consensus**, comment using [this template](https://raw.githubusercontent.com/crowdresearch/collective/master/breaking_consensus). To find out more about this process, read the [how-to](https://github.com/crowdresearch/collective/blob/master/governance/how-to.md).

**Comment by neilthemathguy** [comment]
**BREAKING CONSENSUS** 

Recognition is important and must be done in fair, respectful, and consistent manner whether it is in media, industry, academic communities, requesters/worker communities, or Daemo blog/forum posts. It is important to not lose the past and current contributors’ efforts. 

Above proposal says that ---
> contributors to our community have not been publicly recognized in a way that outlives their significant participation. 

**This is partially true.** Some of the members have already been recognized, for example, in the [WIRED](https://www.wired.com/story/amazons-turker-crowd-has-had-enough/) article, which rest of the collective didn’t know about. This issue needs serious considerations--- it is one of the reasons for broken trust in the community. 

**Regarding nomination for Emeritus**  
Emeritus: (of the former holder of an office, especially a college professor) having retired but allowed to retain their title as an honor.

In the global project, there are several problems with using labels such as “Emeritus”. First, “Emeritus” status has specific meaning associated with it--- it is used in the context of retirement from an office. Daemo is an open source project, not an office, except for individuals who are not volunteers and getting paid/academic credits for their time spent on the project. 

Second, this causation claim is broad and has no relationship with career advancement. In the context of this project, “Emeritus” status does not grant any sort of entitlement and doesn’t indicate anything to advance anyone’s career as compared to the way it is used in academia in general. 

**Needless to say that volunteers should only be marked as inactive, if and only if they indicate that they don’t want to be associated with this project.** Otherwise volunteers can be active or inactive whenever they want. They are voluntarily taking on the responsibilities of Daemo, we should be grateful for that rather than saying  "relieved of ongoing responsibilities”. 

**_We must strive to create an inclusive, respectful, and free environment which attracts volunteers to work for the project._**

*New proposal for recognizing all contributors  Respect and Recognition with the Contributors Page in Daemo #21*

**Comment by markwhiting** [comment]
Thanks for your thoughts, @neilthemathguy.

First, I am happy to concede the recognition aspect to #21. That sounds like a great way to recognize members of this community. I'm interested to see what you make for it. In other words, I think I would be willing to remove point 2 from this proposal because I think your proposal is a more comprehensive approach to that. 

Second, I agree about the potential connotations of 'emeritus'. Do you have any suggestions about how we might frame this kind of status that might not have that problem? The point is to formalize the notion of honorable discharge, i.e. allowing people to speak of their time and achievements in a particular capacity with the Crowd Research Collective and us conferring support for the participation formally.

**Comment by neilthemathguy** [comment]
#21 already covers people's contributions to which they can point to even if they want to stay inactive in the project. 

I would like to know why do you think discharging volunteer contributors is important for the Collective?

**Comment by markwhiting** [comment]
Formalizing 'active' participation is something that we have talked about at length both over the past few months, and at several points prior to this. The aim is not to discharge people unwillingly but more to be able to do things like provide accurate numbers to the media about how many people are involved. ~~, as @shirishgoyal has mentioned in #19.~~

In this case, the goal would be to provide a formal way of doing this that aims to align that shift in status with any other shifts relevant to that person, e.g. responsibilities, duties etc. 

Another way of thinking about this might be as an alumni status.

(edited around updated context from #19)

**Comment by shirishgoyal** [comment]
@markwhiting Just to set the context right, #19 talks about initial influx of people (when any new cohort opens up) who join(ed) the project and didn't(don't) contribute to the project after few days or first month from joining. It doesn't refer to contributors who have been active (and now may be inactive) over a significant period of time.

**Comment by markwhiting** [comment]
OK. My apologies. I will mark my above statement accordingly.

**Comment by neilthemathguy** [comment]
I agree with @shirishgoyal--- both are different things. 

**Open communities must provide equal and inclusive opportunities rather than exploiting its members.** 

There are people here who can afford to stay full time active because they are financially/academically supported. Other people need to take care of their commitments such as jobs, family, business, education, and many other things. 

First, are we recognizing people's efforts? Before recognizing them Emeritus/Alumni/whatever  status, it is important to highlight contributions of the Collective and its members.

**Comment by markwhiting** [comment]
I agree with your overall point here and I agree that I think recognizing peoples efforts is an important step. Hopefully #21 will do that effectively.

I'm happy to wait to see how that pans out over the next week.

**Comment by mbernst** [comment]
I like seeing #21 come back --- thanks!

I am curious how open source communities tend to handle the question of people going inactive for long periods. Doing some digging:

[node.js on liberal contribution](https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951)
>Members can be added to the TC at any time. Any committer can nominate another committer to the TC and the TC uses its standard consensus seeking process to evaluate whether or not to add this new member. Members who do not participate consistently at the level of a majority of the other members are expected to resign.

So, in node.js you are expected to resign from the equivalent of our ops+QR groups if you are not active.

[Opensource.guide on meritocratic OSS](https://opensource.guide/leadership-and-governance/#when-should-i-give-someone-commit-access) (I know we are more liberal contribution than meritocracy):
> It is important to recognise that commitership is a privilege, not a right. That privilege must be earned and once earned it can be removed by the PMC (see next section) in extreme circumstances. However, under normal circumstances committership exists for as long as the committer wishes to continue engaging with the project.

So, in Apache projects you retain rights.

So interesting. I'll keep digging.

**Comment by dmorina** [comment]
Although Daemo is mentioned to be an open source project it definitely doesn't act like one in reality (this whole new process especially) so that argument doesnt really hold, and there are certain roles where you need to "retire" people because if they are not around for a while they lose context and then waste more time by proposing ideas which have been discussed during the time they have been away or teaching them things they might miss, or security concerns and such
